website/docs/r/windows_virtual_machine_scale_set.html.markdown

```diff
@@ -296,6 +296,8 @@ An `extension` block supports the following:
 
 * `auto_upgrade_minor_version` - (Optional) Should the latest version of the Extension be used at Deployment Time, if one is available? This won't auto-update the extension on existing installation. Defaults to `true`.
 
+* `automatic_upgrade_enabled` - (Optional) When Automatic Extension Upgrade is enabled on a VM or scale set, the extension is upgraded automatically whenever the extension publisher releases a new version for that extension. Defaults to `false`.
+
```

Comment:
we'll also need to add this field to both the Linux and Windows VMSS Resources:

* `internal/services/compute/linux_virtual_machine_scale_set_resource.go`
* `internal/services/compute/windows_virtual_machine_scale_set_resource.go`

Reply:
The field is added automatically, because they reuse the same `VirtualMachineScaleSetExtensionsSchema`:

https://github.com/hashicorp/terraform-provider-azurerm/blob/d01b7ad19d19851cd21696b6e6e80ca1e79fd37a/internal/services/compute/linux_virtual_machine_scale_set_resource.go#L144


https://github.com/hashicorp/terraform-provider-azurerm/blob/d01b7ad19d19851cd21696b6e6e80ca1e79fd37a/internal/services/compute/windows_virtual_machine_scale_set_resource.go#L145

Reply:
I would need to copy this function it seems.